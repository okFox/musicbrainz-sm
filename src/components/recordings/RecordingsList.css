ul {
    
    li {
    font-family: 'Acme', sans-serif;
}
}
img {
    border-radius: 5px;
}