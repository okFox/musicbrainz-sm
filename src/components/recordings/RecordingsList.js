import React from 'react';
import PropTypes from 'prop-types';
import RecordingsItem from './RecordingsItem';
import './RecordingsList.css';


const RecordingsList = ({ recordingsArray, artistName }) => {
  const recordings = recordingsArray.map((recording, i) => (

    <li key={i}><RecordingsItem recordingTitle={recording.recordingTitle} artistName={artistName} /></li>

  ));

  return (
    <>
      <section>
        <h1>Recordings</h1>
        <ul>
          {recordings}
        </ul>
      </section>
    </>
  );
};

RecordingsList.propTypes = {
  recordingsArray: PropTypes.array.isRequired,
  artistName: PropTypes.string.isRequired
};

export default RecordingsList;
