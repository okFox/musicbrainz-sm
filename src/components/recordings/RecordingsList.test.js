import React from 'react';
import { shallow } from 'enzyme';
import RecordingsList from './RecordingsList';

describe('RecordingsList component', () => {
  it('renders RecordingsList', () => {
    const wrapper = shallow(<RecordingsList recordingsArray={[]} artistName="name" />);
    expect(wrapper).toMatchSnapshot();
  });
});
