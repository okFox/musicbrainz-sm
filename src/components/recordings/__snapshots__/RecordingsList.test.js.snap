// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecordingsList component renders RecordingsList 1`] = `
<Fragment>
  <section>
    <h1>
      Recordings
    </h1>
    <ul />
  </section>
</Fragment>
`;
